asg-blue.tf

```diff
@@ -0,0 +1,45 @@
+data "aws_ami" "nginx-blue" {
+  most_recent = true
+  owners = ["699330879220"]
+  
+  filter {
+    name   = "root-device-type"
+    values = ["ebs"]
+  }  
+
+  filter {
+    name   = "virtualization-type"
+    values = ["hvm"]
+  }
+  
+  filter {
+    name   = "image-id"
+    values = ["ami-0b8739bd500917e40"]
```

Comment:
Could you use the name of the image instead of the image-id here please ? We want the data source to find the latest available image of the application, not a specific version all the time.